application/database/db.py

```diff
@@ -224,32 +224,73 @@ def add_cre(self, dbcre: CRE):
         if not self.connected:
             return
         self.driver.execute_query(
-            "MERGE (n:CRE {id: $nid, name: $name, description: $description, external_id: $external_id})",
-            nid=dbcre.id,
+            "MERGE (n:CRE {id: $nid, name: $name, description: $description, doctype: $doctype, links: $links, metadata: $metadata, tags: $tags})",
             name=dbcre.name,
+            doctype="CRE",  # dbcre.ntype,
+            nid=dbcre.id,
             description=dbcre.description,
-            external_id=dbcre.external_id,
+            links=[],  # dbcre.links,
+            tags=dbcre.tags,
+            metadata="{}",  # dbcre.metadata,
             database_="neo4j",
         )
 
     @classmethod
     def add_dbnode(self, dbnode: Node):
         if not self.connected:
             return
-        # TODO: Add diffrent Node types
-        self.driver.execute_query(
-            "MERGE (n:Node {id: $nid, name: $name, section: $section, section_id: $section_id, subsection: $subsection, tags: $tags, version: $version, description: $description, ntype: $ntype})",
-            nid=dbnode.id,
-            name=dbnode.name,
-            section=dbnode.section,
-            section_id=dbnode.section_id,
-            subsection=dbnode.subsection or "",
-            tags=dbnode.tags,
-            version=dbnode.version or "",
-            description=dbnode.description,
-            ntype=dbnode.ntype,
-            database_="neo4j",
-        )
+        if dbnode.ntype == "Standard":
+            self.driver.execute_query(
+                "MERGE (n:Standard {id: $nid, name: $name, section: $section, sectionID: $sectionID, subsection: $subsection, tags: $tags, version: $version, description: $description, doctype: $doctype, links: $links, metadata: $metadata, hyperlink: $hyperlink})",
```

Comment:
can we use sqlalchemy here? i'd prefer it if we used an ORM, if it's impossible its ok

Reply:
Can we make this a separate issue that must be handled before we add more queries?  It seems more suitable as a separate PR.